aos-vm.yaml

```diff
@@ -78,10 +76,9 @@ common_data:
     - [AOS_BOOT_IMAGE_VERSION, "%{BUNDLE_IMAGE_VERSION}"]
 
     # Node configuration
-    - [AOS_MAIN_NODE_ID, "%{MAIN_NODE_ID}"]
-    - [AOS_MAIN_NODE_HOSTNAME, "%{MAIN_NODE_ID}"]
-    - [AOS_NODE_GW_IP, "10.0.0.1"]
+    - [AOS_MAIN_NODE_HOSTNAME, "%{MAIN_NODE_HOSTNAME}"]
```

Comment:
Are there duplicates needed? I mean AOS_MAIN_NODE_HOSTNAME & MAIN_NODE_HOSTNAME are basically the same